### PR TITLE
Avoid inline styling

### LINKED ISSUE #117: Avoid inline styles
```
library/Jira/Web/Table/IssuesTable.php:49:                        ['class' => 'small', 'style' => 'display: block'],
library/Jira/Web/Table/IssueDetails.php:138:                    'style' => 'margin-right: 1em; border-radius: 50%;',
library/Jira/Web/Table/IssueDetails.php:173:                ['style' => 'background-color: transparent'],

### PR DESCRIPTION
**Pull request by raviks789**
fixes #117

### CODE REVIEW

File: library/Jira/Web/Table/IssuesTable.php [hunk 1]
```diff
@@ -46,7 +46,7 @@ protected function assemble()
                     $helper->anonymize($issue->fields->summary),
                     Html::tag(
                         'span',
-                        ['class' => 'small', 'style' => 'display: block'],
+                        ['class' => 'small'],
```
Comment (lippserd) [inline library/Jira/Web/Table/IssuesTable.php:49]: I don't like `small`. Any chance to give that class a meaning?

Comment (raviks789) [reply library/Jira/Web/Table/IssuesTable.php]: I have change the class name to `issue-description` as it is used for Jira issue description field.

File: public/css/module.less [hunk 2]
```diff
@@ -72,3 +81,9 @@ img.icon {
   overflow: hidden;
   display: inline-block;
 }
+
+.issue-table {
```
Comment (lippserd) [inline public/css/module.less:85]: Is that wrapping needed?

Comment (lippserd) [reply public/css/module.less:85]: I'm just thinking why `span` and then adding `display: block` :)

Comment (raviks789) [reply public/css/module.less:85]: Now you mention it, I think it makes sense to just use `div`. And remove the styling.

Comment (nilmerg) [reply public/css/module.less]: `p` would be more fitting, if you ask me :sweat_smile:

File: library/Jira/Web/Table/IssuesTable.php [hunk 3]
```diff
@@ -45,8 +45,8 @@ protected function assemble()
                     ),
                     $helper->anonymize($issue->fields->summary),
                     Html::tag(
-                        'span',
-                        ['class' => 'small', 'style' => 'display: block'],
+                        'div',
+                        ['class' => 'issue-description'],
```
Comment (lippserd) [inline library/Jira/Web/Table/IssuesTable.php:49]: Is the class needed at all?

File: public/css/module.less (hunk 2 above)
Comment (raviks789) [reply public/css/module.less]: Changed it to `p`

File: public/css/module.less [hunk 4]
```diff
@@ -21,6 +21,15 @@
     }
 }
 
+.comment {
+    background-color: transparent;
+}
+
+img.comment-author {
```
Comment (lippserd) [inline public/css/module.less:28]: Do we have the class `comment-author` for different tags?

Comment (raviks789) [reply public/css/module.less:28]: No, just wanted to maintain the specificity. Will remove it right away.

**Review by lippserd — APPROVED** [review]

**Review by nilmerg — APPROVED** [review]

**Pull request merged**